web-access-app/src/test/resources/fedora-content-it-servlet.xml

```diff
@@ -19,5 +19,6 @@
     <bean id="fedoraContentService" class="edu.unc.lib.boxc.web.common.services.FedoraContentService">
         <property name="accessControlService" ref="aclService" />
         <property name="repositoryObjectLoader" ref="repositoryObjectLoader" />
+        <property name="client" ref="fcrepoClient" />
```

Comment:
We're also going to need to make this change in all the other context files where fedoraContentService appears, otherwise it probably won't work when deployed to a server.